slp_drawio/tests/unit/parse/transformers/test_parent_calculator_transformer.py

```diff
@@ -0,0 +1,46 @@
+from slp_drawio.slp_drawio.objects.diagram_objects import Diagram, DiagramComponent, DiagramTrustZone
+from slp_drawio.slp_drawio.parse.tranformers.parent_calculator_transformer import ParentCalculatorTransformer
+
+
+class TestParentCalculatorTransformer:
```

Comment:
This class lacks tests to set the parent for:

1. Component has a TrustZone as parent
2. Trustzone has a Component as a parent
3. Trustzone has another TZ as a parent

Reply:
I think these use cases are out of scope of the MVP, but I have done it anyway. I had to modify the DiagramTrustzone class.